src/main/java/net/apnic/rdap/nro/config/NROConfiguration.java

```diff
@@ -4,18 +4,36 @@
 
 import org.springframework.context.annotation.Bean;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.context.annotation.Configuration;
 
 /**
  * Configuration singleton for NRO scraping activities.
  */
 @Configuration
+@ConfigurationProperties(prefix="rdap.scraping.scrapers.nro")
 public class NROConfiguration
 {
-    @ConditionalOnProperty(value="rdap.scraping.scrapers.iana.enabled")
+    String baseURI = null;
+
+    public String getBaseURI()
+    {
+        return baseURI;
+    }
+
+    @ConditionalOnProperty(value="rdap.scraping.scrapers.nro.enabled")
     @Bean(value="nro")
     public NROScraper nroScraper()
     {
+        if(baseURI != null)
```

Comment:
(Suggestion) Is it possible for this to be an empty string? Might be worth checking `!baseURI.isEmpty()`.